### PR TITLE
Add training links resource page

### PR DESCRIPTION
**Pull request by cosden**
This PR adds a new submenu item "Training Links" to the top level "Resources" page. 

In the Education and Training Working Group meeting earlier today we decided that getting an incomplete and imperfect list for now was better than waiting for a more polished product. Especially since it can be expanded and developed in parallel with the rest of the website. So here it is!

For now it is a painfully simple markdown table, with classifications that likely still need some work. The E&T working group will continue to collect and add material, as well as expand functionality (tags, search, sort, etc). 

image

### CODE REVIEW

**Comment by cosden** [comment]
I don't know why the URL checker is failing on this site, https://www.olcf.ornl.gov/cuda-training-series/.

**Comment by cosden** [comment]
Would this be a case of need to add https://www.olcf.ornl.gov to "white_listed_patterns:" in linting.yml?

**Comment by vsoch** [comment]
This looks great! Does "individual links" best describe that table though/ and distinguish from the top one?

File: pages/resources/training.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+These sites have multiple RSE-related links, potentially on a number of topics.
+Rather than reporoduce, we list them here:
+
+| Audience | Style | Domain | Type | Title |
+| -------- | ----- | ------ | ---- | ----- |
+| Beginner | Collection | SE | Tutorial | [Software Carpentry](https://software-carpentry.org/lessons/) |
+| Intermediate | Collection | SE | Links | [INTERSECT](https://intersect-training.github.io/training-links/) |
+| Intermediate |	Collection |	SE | Tutorial |	[The Programming Historian](https://programminghistorian.org/) |
+
+### Individual Links
```
Comment (vsoch) [inline pages/resources/training.md:29]: Maybe just say Lectures and Books?

File: pages/resources/training.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| Beginner | Examples | CS | Lecture | [Intro to Unix](https://vimeo.com/279273125) |
+| Beginner | Examples | SE | Lecture | [Intro to Vim](https://vimeo.com/279277260) |
+| Beginner | Examples | SE | Lecture | [Intro to Git](https://vimeo.com/279287047) |
+| Beginner | Visual | CS | Lecture | [Intro to Fortran](https://vimeo.com/279286109) |
+| Beginner | Visual | CS | Lecture | [Intro to C](https://vimeo.com/279284053) |
+| Beginner | Examples | HPC/Parallel Computing | Lecture | [Intro to Batch Scheduler](https://vimeo.com/279278776) |
+| Intermediate | Visual | HPC/Parallel Computing | Lecture | [Intro to Parallel Computing](https://vimeo.com/279288481) |
+| Intermediate | Examples | HPC/Parallel Computing | Lecture | [Intro to MPI](https://vimeo.com/279313080) |
+| Intermediate | Auditory | HPC/Parallel Computing | Lecture | [Nvidia Basics](https://vimeo.com/371964454) |
+| Intermediate | Examples | HPC/Parallel Computing | Lecture | [Cuda Tutorial](https://www.olcf.ornl.gov/cuda-training-series/) |
```
Comment (mtbc) [inline pages/resources/training.md:43]:
```suggestion
| Intermediate | Examples | HPC/Parallel Computing | Lecture | [CUDA Tutorial](https://www.olcf.ornl.gov/cuda-training-series/) |
```

File: pages/resources/training.md (hunk 1 above)
Comment (cosden) [reply pages/resources/training.md:29]: Our classification system is a work in progress and is almost certainly going to change, but I like your suggestion, let me propose a slight change to both high-level classifications.

**Review by vsoch — APPROVED** [review]

**Pull request merged**